src/main/resources/resources/languages/languages_it_IT.properties

```diff
@@ -87,3 +90,9 @@ panel.save=Salva
 panel.open=Apri
 panel.saveModifiedQuestionMessage=Il file attuale è stato modificato.\nVuoi salvare le modifiche prima della chiusura?
 panel.unencryptedDataWarningMessage=Nota che tutti i dati verranno archiviati non crittografati.\nAssicurati di conservare il file esportato in un luogo sicuro.
+
+language.languageChanged=Language has changed successfully.
```

Comment:
@bovirus If you have some time, can you please suggest the correct translation to Italian for these strings added?

Reply:
Thanks for the input, @bovirus . Can you also please check the other new strings added to this file?

Reply:
@edimoral 

Where can I get the latest Italian language (i see until line 89)?